specification/network/resource-manager/Microsoft.Network/stable/2021-08-01/firewallPolicy.json

```diff
@@ -2139,15 +2121,5 @@
         }
       }
     }
-  },
-  "parameters": {
-    "ApiVersionParameterMinimumPremium": {
-      "name": "api-version",
-      "in": "query",
-      "required": true,
-      "type": "string",
-      "default": "2020-04-01",
```

Comment:
It only needs to remove this default value. Can you reference the api version parameter from common-types rather than have it defined in firewallPolicy.json.

Reply:
Switched to the common api-version type

Reply:
@raych1  can you please review it now?